swift/ql/lib/codeql/swift/dataflow/internal/DataFlowDispatch.qll

```diff
@@ -57,17 +61,78 @@ class DataFlowCallable extends TDataFlowCallable {
 
   /** Gets the location of this callable. */
   Location getLocation() { result = func.getLocation() }
+
+  AbstractFunctionDecl getUnderlyingCallable() { result = func }
```

Comment:
Should this return something compatible with closures?

Reply:
Yeah, it probably should. How about we add `ClosureExpr` to https://github.com/github/codeql/blob/main/swift/ql/lib/codeql/swift/controlflow/internal/Scope.qll#L5 and use `Callable::TypeRange` instead?

Reply:
https://github.com/github/codeql/pull/9557 makes the required changes for this to work. Once that PR is merged I'll fix this comment.

Reply:
Fixed in eff046e2f75f6ab8ef48029ddbae97f95bad9504.